spec/facility_spec.rb

```diff
@@ -23,4 +24,62 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  describe 'add services to facility' do
+    before(:each) do
+      @facility_1 = Facility.new({name: 'DMV Tremont Branch', address: '2855 Tremont Place Suite 118 Denver CO 80205', phone: '(720) 865-4600'})
+      @facility_2 = Facility.new({name: 'DMV Northeast Branch', address: '4685 Peoria Street Suite 101 Denver CO 80239', phone: '(720) 865-4600'})
+      @cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+      @bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+      @camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} )
+      end
+
+    it 'can add services to a facility' do
+      expect(@facility_1.add_service('Vehicle Registration')).to eq(["Vehicle Registration"])
+    end
+
+    it 'can read registration date' do
+      expect(@cruz.registration_date).to eq(nil)
+    end
+
+    it 'can hold a list of registered vehicles' do
+      expect(@facility_1.registered_vehicles).to eq([])
+    end
+
+    it 'has collected fees default to 0' do
+      expect(@facility_1.collected_fees).to eq(0)
+    end
+
+    it 'can register a vehicle' do
+      @facility_1.add_service("Vehicle Registration")
+      @facility_1.register_vehicle(@cruz)
+      expect(@cruz.registration_date).to eq(Date.today)
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@facility_1.registered_vehicles).to include(@cruz)
```

Comment:
This whole test example block is great! For this line, you could be even more specific and `expect(@facility_1.registered_vehicles).to eq([@cruz])`, verifying that it is the only vehicle in the array